client/src/backend/components/analytics/parts/chartOptions.js

```diff
@@ -44,7 +54,7 @@ const rechartsOptions = ({ tooltipLabel }) => ({
     type: "number",
     scale: "time",
     stroke: AXIS_COLOR,
-    tickFormatter: value => formatDate(value, "MM/dd"),
+    tickFormatter: value => formatDate(value, "MM/dd"), // TODO: Localize date ticks, refactor required
```

Comment:
@1aurend from the l10n work you did with other dates in the app, can you think of an easy way to localize these x-axis labels?
<img width="623" alt="Screen Shot 2022-04-25 at 11 09 31 AM" src="https://user-images.githubusercontent.com/12454203/165149135-05c4d9d1-ae0a-4bd4-84e6-75d213880864.png">

Reply:
Just pushed a solution for this and couple other little l10n things in the chart.